layered-architectures-module-2/Catalog.WebUI/Controllers/ProductsController.cs

```diff
@@ -14,6 +15,13 @@ public async Task<ActionResult<ProductsVm>> GetProducts(int? id = null)
         var query = new GetProductsQuery() { Id = id };
         return await Mediator.Send(query);
     }
+    
+    [HttpGet("paginated")]
```

Comment:
very poor name. Just GET without an `id` could be enough

Reply:
Fixed